### DISCUSSION TITLE
npx prisma migrate dev: sqlite command not found

### DISCUSSION THREAD

**Discussion by lolove** [prisma/prisma#19601] [Q&A]
### Bug description

When I tried to run `npx prisma migrate dev` with SQLite, it always said '/bin/sh: sqlite: command not found' (as below). I installed sqlite3 and add the alias(sqlite->sqlite3). I am wondering didn't I miss any steps?

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: Generator "sqlite" failed:

/bin/sh: sqlite: command not found
```

### How to reproduce

1. npm install prisma --save-dev
2. npm install @prisma/client
3. npx prisma init
4. Add schema.prisma
5. run `npx prisma migrate dev`
6. See Error

I worked with this tutorial -> https://www.howtographql.com/graphql-js/4-adding-a-database/

### Expected behavior

Correctly finish `npx prisma migrate dev` or `npx prisma generate`.

### Prisma information

```prisma
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: Generator "sqlite" failed:

/bin/sh: sqlite: command not found
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: macOS 13.3
- Database: SQLite
- Node.js version: v16.20.0

### Prisma Version

```
prisma                  : 4.15.0
@prisma/client          : 4.15.0
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Migration Engine        : migration-engine-cli 8fbc245156db7124f997f4cecdd8d1219e360944 (at node_modules/@prisma/engines/migration-engine-darwin-arm64)
Format Wasm             : @prisma/prisma-fmt-wasm 4.15.0-28.8fbc245156db7124f997f4cecdd8d1219e360944
Default Engines Hash    : 8fbc245156db7124f997f4cecdd8d1219e360944
Studio                  : 0.484.0
```

**Comment by janpio** [comment]
Did you maybe put `sqlite` in the `provider` field of the `generator` block instead of the `datasource`?

**Reply by ludralph** [reply]
Hi @lolove, It will be helpful if you can share your schema.prisma file

**Reply by nurul3101** [reply]
@lolove We are closing this Discussion due to inactivity. Please open a new Discussion if you have the a similar problem :pray:
